packages/playground/website/src/components/site-manager/site-info-view/style.module.css

```diff
@@ -1,40 +1,62 @@
 /* Site details */
 
-.site-manager-site-info-container {
-	/* Black & White / White */
+.site-info-view {
+	--padding-size: 24px;
+	--color-gray-600: #6a6a6a;
+	--color-gray-900: #1e1e1e;
```

Comment:
@adamziel thanks for this good example of
- making use of scoped CSS properties
- using shorter, meaningful class names within CSS modules because they effectively give us an isolated namespace